projects/Generic/linux/linux.x86_64.conf

```diff
@@ -1866,7 +1871,7 @@ CONFIG_WIREGUARD=m
 CONFIG_MACVLAN=m
 # CONFIG_MACVTAP is not set
 # CONFIG_IPVLAN is not set
-# CONFIG_VXLAN is not set
```

Comment:
And this is how it started, enabling VXLAN overlay networking support in the kernel.